wallet/subnet/primary/examples/create-chain/main.go

```diff
@@ -5,34 +5,45 @@ package main
 
 import (
 	"context"
-	"encoding/hex"
 	"log"
+	"math"
 	"time"
 
 	"github.com/ava-labs/avalanchego/genesis"
 	"github.com/ava-labs/avalanchego/ids"
 	"github.com/ava-labs/avalanchego/utils/set"
+	"github.com/ava-labs/avalanchego/vms/example/xsvm"
 	"github.com/ava-labs/avalanchego/vms/secp256k1fx"
 	"github.com/ava-labs/avalanchego/wallet/subnet/primary"
+
+	xsgenesis "github.com/ava-labs/avalanchego/vms/example/xsvm/genesis"
 )
 
 func main() {
 	key := genesis.EWOQKey
 	uri := primary.LocalAPIURI
 	kc := secp256k1fx.NewKeychain(key)
 	subnetIDStr := "29uVeLPJB1eQJkzRemU8g8wZDw5uJRqpab5U2mX9euieVwiEbL"
-	genesisHex := "00000000000000000000000000017b5490493f8a2fff444ac8b54e27b3339d7c60dcffffffffffffffff"
-	vmID := ids.ID{'x', 's', 'v', 'm'}
+	genesis := &xsgenesis.Genesis{
+		Timestamp: time.Now().Unix(),
+		Allocations: []xsgenesis.Allocation{
+			{
+				Address: genesis.EWOQKey.Address(),
+				Balance: math.MaxUint64,
+			},
+		},
+	}
+	vmID := xsvm.ID
```

Comment:
This example was included before the `xsvm` was in `avalanchego`. I think this makes a lot more sense than some random hex and a random `ID`.